src/index.ts

```diff
@@ -130,7 +135,7 @@ function preactPlugin({
 							: "@babel/plugin-transform-react-jsx-development",
 						{
 							runtime: "automatic",
-							importSource: "preact",
+							importSource: jsxImportSource ?? "preact",
```

Comment:
nullish coalescing while having no benefit over `||` operator in this case, still shouldn't break anyone's environment, as it's supported in Node.js 14 (oldest version that hasn't reach EOL)